libs/sdk-backend-tiger/src/backend/organization/notifications.ts

```diff
@@ -0,0 +1,71 @@
+// (C) 2024 GoodData Corporation
+
+import { ITigerClient } from "@gooddata/api-client-tiger";
+import { INotification } from "@gooddata/sdk-model";
+import {
+    INotificationsQuery,
+    IOrganizationNotificationService,
+    UnexpectedError,
+} from "@gooddata/sdk-backend-spi";
+
+import { TigerAuthenticatedCallGuard } from "../../types/index.js";
+import { convertNotificationFromBackend } from "../../convertors/fromBackend/NotificationsConvertor.js";
+import { NotificationsQuery } from "./notificationsQuery.js";
+
+export class OrganizationNotificationService implements IOrganizationNotificationService {
+    constructor(public readonly authCall: TigerAuthenticatedCallGuard) {}
+
+    /**
+     * Get notification by id
+     * @param id - id of the notification
+     * @returns Promise resolved with notification.
+     *
+     * @beta
+     */
+    public getNotification(id: string): Promise<INotification> {
```

Comment:
I'm wondering why there's no api for getting notification by id. Also with this approach, be careful about pagination, it may happen that the notification with supplied id will be out of page (/apidocs states now the default of 20)

Reply:
method removed